CONTRIBUTING.md

```diff
@@ -72,3 +72,14 @@ OR
 * The module is under the Node.js foundation Github org
 OR
 * The module is identified as an important module by a Node.js Working Group
+
+#### Procedure
+
+After making sure you adhere to the above requirements, do the
+following:
+
+1. Add the module to
```

Comment:
I guess we might as well be exhaustive:

1. `git clone https://github.com/nodejs/citgm.git && cd citgm`

This also (hopefully) makes it clear that you're running `npm link` in the citgm directory, not in your module directory.